db/seeds.rb

```diff
@@ -5,3 +5,46 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Category.delete_all
+
+categories = ["beach", "park", "playground"]
+
+categories.each do |category|
+  Category.create(name: category)
+end
+
+Playspace.delete_all
+
+playspace_list = [
```

Comment:
As a rule of thumb, when you have lots of data to insert in the seed, you should do explictly include the column names:

```rb
Playspace.create(name: "Hong Kong Park", 
  address: "19 Cotton Tree Drive", 
  district: "Central", 
  region: "HK", 
  bike_path: nil, 
  merry_go_round: nil, 
  play_structure: true,
  ...
  category_id: Category.find_by name: "beach"
)

```